### PR TITLE
feat(PageHero)!: add short text variation

### PR DESCRIPTION
**Pull request by aresnik11**
### Overview

Adds a `textLength` prop to `PageHero` for long text (9 col text, 3 col image) vs short text (6 col text, 6 col image). Videos are set up specifically to have 7 cols text and 5 cols for video so that was left as is. Also refactors to use more built in gamut system props.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/J5yB1jyELiNt3Vb47T9GcE/Job-Readiness?node-id=2%3A43
- [x] Related to JIRA ticket:[REACH-1323]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.90-alpha.c24f19.0<br/>@codecademy/gamut-labs@23.0.1-alpha.c24f19.0<br/>@codecademy/styleguide@46.0.1-alpha.c24f19.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.92-alpha.7da6ee.0<br/>@codecademy/gamut-labs@23.2.0-alpha.7da6ee.0<br/>@codecademy/styleguide@46.3.0-alpha.7da6ee.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://614e2422a7cc9b2a7038251a--gamut-preview.netlify.app>https://614e2422a7cc9b2a7038251a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/614e2422a7cc9b2a7038251a>Deploy Logs</a></p>

File: packages/gamut-labs/src/LandingPage/PageHero.tsx [hunk 1]
```diff
@@ -42,52 +39,58 @@ export type PageHeroProps = BaseProps & {
...
+    right: 12,
+  };
 };
 
 export const PageHero: React.FC<PageHeroProps> = ({
   title,
   desc,
   cta,
   media,
+  textLength,
```
Comment (christian-dinh) [inline packages/gamut-labs/src/LandingPage/PageHero.tsx:84]: should this default to `'long'` so current consumers don't need to change at all?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.92-alpha.0a1e7f.0<br/>@codecademy/gamut-labs@23.2.0-alpha.0a1e7f.0<br/>@codecademy/styleguide@46.3.0-alpha.0a1e7f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61548f597b0b4c2fd30f6ef5--gamut-preview.netlify.app>https://61548f597b0b4c2fd30f6ef5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61548f597b0b4c2fd30f6ef5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.94-alpha.2f0767.0<br/>@codecademy/gamut-labs@23.3.1-alpha.2f0767.0<br/>@codecademy/styleguide@46.4.1-alpha.2f0767.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6154940a38d3fa310db3296a--gamut-preview.netlify.app>https://6154940a38d3fa310db3296a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6154940a38d3fa310db3296a>Deploy Logs</a></p>

**Review by christian-dinh — APPROVED** [review]
ty!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.95-alpha.8afe9e.0<br/>@codecademy/gamut-labs@23.3.2-alpha.8afe9e.0<br/>@codecademy/styleguide@46.4.2-alpha.8afe9e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61549daba3fd4500d09afe9d--gamut-preview.netlify.app>https://61549daba3fd4500d09afe9d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61549daba3fd4500d09afe9d>Deploy Logs</a></p>

**Review by judahtanthony — APPROVED** [review]
🚀

**Pull request merged**